app/controllers/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
```

Comment:
Can be removed - Devise handles signups for us. You should use an `after_create` hook in `app/models/user.rb` to send the email.